examples/skia-demo-player-c/main.cpp

```diff
@@ -0,0 +1,178 @@
+#include <X11/Xlib.h>
```

Comment:
Relying on the X Window System and Skia is a bad choice for portability. How about using SDL2 instead?

Reply:
Updated the examples to use SDL2